### PR TITLE
Create 2022-JAN-31 file

### PR DESCRIPTION
**Pull request by sarabeth**
Release Notes for Compliance Version 2.0

### CODE REVIEW

File: release-notes/2022-JAN-31.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- The Compliance App look-and-feel is updated and reflects a refreshed organizational flow.
+![](../assets/comv2.png)
+ 
+- Actions taken in the Compliance app are now driven by a new left navigation bar and organizes actions into two camps: Frameworks and tertiary actions.
+
+- All controls are now located in a central repository called the Controls Library, allowing for controls to be reused between frameworks and benchmarks. 
+
+- Filtering by value with a compliance framework is now supported, allowing for faster navigation to compliance gaps, evidence, and applicable controls. 
+
+- Frameworks are now broken out into separate views, allowing for better navigation.
```
Comment (AnneNielsenJ1) [inline release-notes/2022-JAN-31.md:16]: Might be worth listing that frameworks are broken out into Benchmarks, Questionnaires, etc.

File: release-notes/2022-JAN-31.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# JupiterOne 31 January 2022 Compliance Version 2.0 Release
+## January 31, 2022
+
+## New Features and Improvements
+Compliance Version 2.0 is live and includes major updates, including the following:
```
Comment (AnneNielsenJ1) [inline release-notes/2022-JAN-31.md:5]: You are using "includes" twice in the sentence. Perhaps something like "Compliance Version 2.0 is now live and available in JupiterOne today! This update includes the following enhancements:"

**Review by AnneNielsenJ1 — COMMENTED** [review]
Overall looks good! Just a couple of suggestions to consider.

File: release-notes/2022-JAN-31.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## New Features and Improvements
+Compliance Version 2.0 is now live and available in JupiterOne today! This update includes the following enhancements:
+
+- The look-and-feel for Compliance is updated and reflects a refreshed organizational flow.
+![](../assets/comv2.png)
+ 
+- Actions taken within Compliance are now driven by a new left navigation bar and organizes actions into two camps: Frameworks and tertiary actions.
+
+- All controls are now located in a central repository called the Controls Library, allowing for controls to be reused between frameworks and benchmarks.
```
Comment (janettelynch) [inline release-notes/2022-JAN-31.md]: Nowhere in the UI is there a title of Controls Library, so I would suggest just calling it a controls library.

**Pull request merged**